web-ui/src/main/resources/catalog/templates/signin.html

```diff
@@ -8,7 +8,7 @@ <h1 data-translate="">loginTitle</h1>
         <div class="panel-body">
           <p data-translate="">loginText</p>
           <form class="form-horizontal" name="gnSigninForm" action="{{formAction}}"
-                method="post" role="form" data-ng-if="::user" data-ng-show="!shibbolethEnabled">
+                method="post" role="form" data-ng-if="::user" data-ng-show="!shibbolethHideLogin">
```

Comment:
Would not be better to check both `shibbolethEnabled` and `shibbolethHideLogin`?